modular_skyrat/modules/New_drinks/code/modules/food_and_drinks/recipes/drink_recipes.dm

```diff
@@ -0,0 +1,8 @@
+
+/datum/chemical_reaction/cum_in_a_hot_tub
+	results = list(/datum/reagent/consumable/ethanol/cum_in_a_hot_tub = 3)
+	required_reagents = list(/datum/reagent/consumable/ethanol/vodka = 2, /datum/reagent/consumable/ethanol/white_russian = 1, /datum/reagent/consumable/ethanol/irish_cream = 0.1)
```

Comment:
I'm not using a code suggestion because I want my GitHub to be free of NSFW-related commits, but I suggest making this 1 unit of irish cream instead of 0.1, otherwise it's going to be really annoying for bartenders unless they do it in large quantities. Using floating point numbers is also not the best of ideas when it comes to making chemical reactions, because you're going to end up leaving decimals of irish cream at the end if you don't go with a perfect ratio of 10x that recipe.